integrations/lib/multiwoven/integrations/source/aws_athena/client.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require "aws-sdk-athena"
+
+module Multiwoven::Integrations::Source
+  module AWSAthena
```

Comment:
While naming the class or module we follow the camel case naming convention based on the folder naming. If the folder name is `aws_athena` then the module name will be `AwsAthena`. Please update this accordingly